tests/create-job.executor.test.ts

```
@@ -0,0 +1,1099 @@
+// ---------------------------------------------------------------------------
+
+describe("CreateJob Executor — Happy Path", () => {
+  // Dynamically import executor so it picks up the test seam resets
+  // We need to re-import for each test because the executor reads
+  // the service overrides at call time (via the test seams).
+
+  test("VAL-CJ-BG-001: job transitions from pending to generating", async () => {
+    installHappyPathMocks();
+    const { executeCreateJob } = await import(
+      "../src/api/v2/agent-templates/services/job-executor"
+    );
```

ESLint

[error] Replace `·await·import(⏎······"../src/api/v2/agent-templates/services/job-executor"⏎····` with `⏎······await·import("../src/api/v2/agent-templates/services/job-executor"`

(prettier/prettier)